### DISCUSSION TITLE
Cypress OPENSSL_internal:NO_START_LINE

### DISCUSSION THREAD

**Discussion by shahrid92** [cypress-io/cypress#15243] [Questions and Help] [answered]
Anyone know this error? Already did cypress cache clear then reinstall it again but still getting this error. couldn't find any solution to fix this issue.

```
Error: error:0900006e:PEM routines:OPENSSL_internal:NO_START_LINE
```
```
{
  library: 'PEM routines',
  function: 'OPENSSL_internal',
  reason: 'NO_START_LINE',
  code: 'ERR_OSSL_PEM_NO_START_LINE'
}
```

Version : 6.5.0

**Comment by nitsan** [comment, marked as answer]
Try to add `XDG_CONFIG_HOME` to your run command: 
```
# assuming these are somehow run simultaneously
XDG_CONFIG_HOME=/tmp/cyhome1 cypress run...
XDG_CONFIG_HOME=/tmp/cyhome2 cypress run...
```
from: https://www.gitmemory.com/issue/cypress-io/cypress/8705/720226424

**Comment by Sam152** [comment]
I ran into this issue running cypress on OSX. In my case the fix was as follows:

1. Navigate into the directory cypress stores it's proxy certificates. In my case this folder was located here: 
```
cd ~/Library/Application\ Support/Cypress/cy/production/proxy/
```
2. Within this folder, there is `certs` and `keys`, both have files with a list of domains accessed during cypress tests. In my case the final two files in each folder were empty:
```
~#: ls -lah key
total 240
...
-rw-r--r--   1 sam  staff     0B  6 May 16:40 www.google-analytics.com.key
-rw-r--r--   1 sam  staff     0B  6 May 16:40 www.google-analytics.com.public.key

~#: ls -lah certs
total 120
...
-rw-r--r--   1 sam  staff     0B  6 May 13:27 www.google-analytics.com.pem
-rw-r--r--   1 sam  staff     0B  6 May 13:27 www.googletagmanager.com.pem
```
3. I'm not sure why these `.pem` and `.key` files were empty for a handful of domains, but deleting them and rerunning a cypress test successfully regenerated them as non-empty and the tests started to pass.
